packages/bus-query-operation/lib/ActorQueryOperation.ts

```diff
@@ -177,55 +183,6 @@ export abstract class ActorQueryOperation extends Actor<IActionQueryOperation, I
   }
 }
 
-export interface IExpressionContext {
-  now?: Date;
-  baseIRI?: string;
-  // Exists?: (expr: Algebra.ExistenceExpression, bindings: Bindings) => Promise<boolean>;
-  // bnode?: (input?: string) => Promise<RDF.BlankNode>;
-}
-
-export interface IActionQueryOperation extends IAction {
```

Comment:
As to not require making this a breaking change, I would re-export the imported types here.

Reply:
Ah yes, I will put those back in. I had dropped them to get the TS compiler to help me refactor stuff internally but not putting them back would likely break other packages depending on Comunica.

Reply:
This should be resolved now. I've opted to rename types using the `T` prefix instead of the `I` prefix in the new `@comunica/types` package (feels more coherent) but re-exported them without any prefix to maintain backward compatibility.

Reply:
> I've opted to rename types using the T prefix instead of the I prefix in the new @comunica/types package

While I'm not opposed to the convention of prefixing types with `T`, such a change will have far-stretching consequences. If we want to be consistent, we'd have to prefix _all_ types in this repo with `T`.
Also, this may confuse users of the current types (without `T`).

For these reasons, I would suggest omitting the `T` prefix.

Reply:
No objections on my end. I thought that, given we're adding a new package, it might be a good idea to adopt a less confusing (IMHO) naming convention right from the start. But, I do understand the argument for consistency. I'll rename without the `T`.

Reply:
Done!